qa/suites/upgrade/client-upgrade-jewel/jewel-client-x/basic/1-install/jewel-client-x.yaml

```diff
@@ -1,7 +1,7 @@
 tasks:
 - install:
     branch: jewel 
-    exclude_packages: ['ceph-mgr','libcephfs2','libcephfs-devel','libcephfs-dev']
+    exclude_packages: ['ceph-mgr','libcephfs2','libcephfs-devel','libcephfs-dev','python34-cephfs','python34-rados']
```

Comment:
shall we s/python34/python36/ here?

Reply:
@tchaikov my guess is that we may need python34 to be used for `jewel`

Reply:
strange enough, we don't install any python3 package back in jewel. we started testing python3 since https://github.com/ceph/ceph/pull/21948 and that PR was not backported to jewel. it was backported to luminous though.

Reply:
so, @yuriw what issue you are trying to address here? could you paste a link of the test failure ?

Reply:
also, to my best understanding, python3 is not supported by jewel.

Reply:
@tchaikov I am not sure if I have the run link ATM.  But the idea is similar as in https://github.com/ceph/ceph/pull/27910 , when we want to test new client on old cluster, `jewel` in this case.  

Re: "python3 is not supported by jewel." we are excluding them in order to install `jewel` in teh first step.

Reply:
@yuriw we started to build python36-* packages since https://github.com/ceph/ceph-build/commit/100a0fb51f547856433af8ff8e28d1d5e769e82d, and in https://github.com/ceph/teuthology/pull/1276/commits/3bafacf3ee113611eb1d5d8aa73fa932af271331, we normalize all python3-* packages to python36-* when installing them. the list of packages to install comes from https://github.com/ceph/ceph/blob/master/qa/packages/packages.yaml

so i think, if we want to add python3 packages into the exclude list, we need to put python36-* instead of python34-*.

Reply:
@tchaikov see your point I want to test https://github.com/ceph/ceph/pull/27936 before merging/fixing  this